.flowconfig

```diff
@@ -0,0 +1,47 @@
+[include]
+
+[ignore]
+# We fork some components by platform. This gives duplicate provider
+# errors if we don't ignore. From base react-native .flowconfig
+.*/*[.]android.js
+
+# Additional libraries we need to ignore because of flow errors in
+# the libraries.
+.*/node_modules/animated/.*
+.*/node_modules/react-native-web/.*
+.*/node_modules/react-native-windows/.*
```

Comment:
will this fail if the directory isn't there? we had issues running windows yesterday.